src/motor_interface.hpp

```diff
@@ -21,25 +22,49 @@ class MotorInterface {
   public:
     ///< Enum class with all available motor interfaces
     enum class INTERFACE { DC, STEPPER };
+    ///< Enum class with all possible motor directions
+    enum class DIRECTION { FORWARD, BACKWARD };
 
   private:
     ///< The interface that is currently selected
     INTERFACE selectedInterface = INTERFACE::DC;
 
+    ///< The direction of the motor rotation, forward by default
+    DIRECTION direction = DIRECTION::FORWARD;
+    ///< Motor rotation speed, 0 - 100%
+    uint8_t speed = 0;
+
   public:
     /**
      * @brief Default constructor
      */
     MotorInterface();
+
     /**
      * @brief getter for selectedInterface

```

Comment:
Check wiki for proper doxygen documentation, you need to document the return value. This goes for the doxygen of other functions as well.